pages/common/pv.md

```diff
@@ -7,9 +7,9 @@
 
 `pv {{path/to/file}}`
 
-- Measure the speed and amount of data flow between pipes (`-s` is optional):
+- Measure the speed and amount of data flow between pipes (`--size` is optional):
 
-`command1 | pv -s {{expected_amount_of_data_for_eta}} | command2`
+`command1 | pv --size {{expected_amount_of_data_for_eta}} | command2`
```

Comment:
I'm not convinced that specifying long options here is needed.

Reply:
I agree, I only changed it because of 3 in CONTRIBUTING.md. 

I also think that `-Ss` is more elegant than `--stop-at --size`. Do you think I should change them both?  Having one in short form and the other in long feels wrong, too.

Reply:
> I also think that -Ss is more elegant than --stop-at --size.

It's faster to type, I agree. But it's unreadable: it's unclear how many options are presented for a random person: two `-S` & `-s` or one `-S` with value `s` (because different programs have different styles of writing options). 😄 If we want to present both long and short options we should decide how to do that and write about our style guide.

Personally I don't like joining options together and consider it's a legacy unreadable option format which introduces ambiguity.